code/datums/traits/neutral.dm

```diff
@@ -10,7 +10,13 @@
 	lose_text = "<span class='notice'>You can taste again!</span>"
 	medical_record_text = "Patient suffers from ageusia and is incapable of tasting food or reagents."
 
-
+/datum/trait/gay
+	name = "Homosexual"
+	desc = "You prefer people of your own gender."
+	value = 0
+	mob_trait = TRAIT_HOMOSEXUAL
+	gain_text = "<span class='notice'>You suddenly feel like you prefer your own gender.</span>"
+	lose_text = "<span class='notice'>You feel like you prefer other genders again.</span>"
```

Comment:
If this is a trait that can selected in character preferences this line doesn't really make sense as it implies that the character was originally straight.  Should probably be 
`lose_text = "<span class='notice'>You feel like you prefer other genders.</span>" `